tensorflow/buildspec.yml

```diff
@@ -24,7 +24,7 @@ images:
   BuildTensorflowGPUJobsPy3DockerImage:
     <<: *JOBS_REPOSITORY
     build: &TENSORFLOW_GPU_JOBS_PY3 false
-    image_size_baseline: 8014
+    image_size_baseline: 9400
```

Comment:
Why did it go up?

Reply:
We were close to the limit before and the new update grew the container.Seems the base container grew in size.

Reply:
I don't mean to harp on this, but 1.4 GB is a lot for the base container to grow. Do you know what it was?

Reply:
Limited the size to more closely follow the build